package.json

```
@@ -92,6 +93,7 @@
     "lib/"
   ],
   "scripts": {
+    "format": "prettier --write \"**/*.js\"",
```

```suggestion
    "format": "prettier \"**/*.js\"",
    "format:fix": "prettier --write \"**/*.js\"",
```
As it is suggested to use `npm run format` in the ci, I expect that that command only checks if the format is valid. Any modification in code done via script should be done in a different command like `npm run format:fix`